test/events/failIssueRes.json

```diff
@@ -0,0 +1,391 @@
+{
```

Comment:
I know you just copied the responses for each of these directly from what the API sent but we likely want to clean up all these payloads to just include the bare minimum we need for our tests. That makes cleaning it up and understanding how the tests are running much easier in the long run.